pkg/apis/autoscaling/v1alpha1/hvpa_types.go

```diff
@@ -32,18 +32,43 @@ type VpaTemplateSpec struct {
 	// for individual containers. If not specified, the autoscaler computes recommended
 	// resources for all containers in the pod, without additional constraints.
 	// +optional
-	ResourcePolicy *vpa_api.PodResourcePolicy `json:"resourcePolicy,omitempty" protobuf:"bytes,3,opt,name=resourcePolicy"`
+	ResourcePolicy *vpa_api.PodResourcePolicy `json:"resourcePolicy,omitempty" protobuf:"bytes,2,opt,name=resourcePolicy"`
 }
 
+// UpdatePolicy describes the rules on how changes are applied.
+type UpdatePolicy struct {
+	// Controls when autoscaler applies changes to the resources.
+	// The default is 'On'.
+	// +optional
+	UpdateMode *string `json:"updateMode,omitempty" protobuf:"bytes,1,opt,name=updateMode"`
+}
+
+const (
+	// UpdateModePurge means that HPA/VPA will not be created.
+	UpdateModePurge string = "Purge"
+	// UpdateModeOff means that autoscaler never changes resources.
+	UpdateModeOff string = "Off"
+	// UpdateModeOn means that autoscaler can update resources during the lifetime of the resource.
+	UpdateModeOn string = "On"
```

Comment:
```suggestion
	UpdateModeAuto string = "Auto"
```

Any reason not be consistent with VPA? Also we can think of supporting `Initial` also in the future.